xlsform_app/views.py

```diff
@@ -18,7 +18,7 @@
 DJANGO_TMP_HOME = os.environ['DJANGO_TMP_HOME']
 
 class UploadFileForm(forms.Form):
-    file = forms.FileField()
+    file = forms.FileField(attrs={ 'accept': '.xls, .xlsx' })
```

Comment:
You made this change to Central, and I've been a fan! However, when I checked this out locally, I saw an error:

```
  File "xlsform-online/xlsform_app/views.py", line 21, in UploadFileForm
    file = forms.FileField(attrs={ 'accept': '.xls, .xlsx' })
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/django/forms/fields.py", line 562, in __init__
    super().__init__(**kwargs)
TypeError: Field.__init__() got an unexpected keyword argument 'attrs'
```

I did a quick search, but I don't see an extremely simple option along these lines. Possibly you pass a `FileInput` as a `widget` to the `FileField` and specify `accept` on the `FileInput`?

Reply:
Thanks for testing!  It would be great to be able to run this project locally.  :wave: https://github.com/getodk/xlsform-online/issues/30